python/lsst/afw/geom/utils.py

```diff
@@ -25,19 +24,23 @@
 
 In the case of the assert functions, importing them makes them available in lsst.utils.tests.TestCase
 """
+__all__ = ["assertAnglesAlmostEqual", "assertPairsAlmostEqual",
```

Comment:
PEP-8 says the opposite. https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

I think there was a recent discussion on this topic with the style guide.